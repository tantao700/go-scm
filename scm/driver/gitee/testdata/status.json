{
    "created_at": "2012-07-20T01:19:13Z",
    "updated_at": "2012-07-20T01:19:13Z",
    "state": "success",
    "target_url": "https://ci.example.com/1000/output",
    "description": "Build has completed successfully",
    "id": 1,
    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/statuses/6dcb09b5b57875f334f61aebed695e2e4193db5e",
    "context": "continuous-integration/drone",
    "creator": {
        "login": "octocat",
        "id": 1,
        "avatar_url": "https://github.com/images/error/octocat_happy.gif",
        "gravatar_id": "",
        "url": "https://gitee.com/api/v5/users/octocat",
        "html_url": "https://github.com/octocat",
        "followers_url": "https://gitee.com/api/v5/users/octocat/followers",
        "following_url": "https://gitee.com/api/v5/users/octocat/following{/other_user}",
        "gists_url": "https://gitee.com/api/v5/users/octocat/gists{/gist_id}",
        "starred_url": "https://gitee.com/api/v5/users/octocat/starred{/owner}{/repo}",
        "subscriptions_url": "https://gitee.com/api/v5/users/octocat/subscriptions",
        "organizations_url": "https://gitee.com/api/v5/users/octocat/orgs",
        "repos_url": "https://gitee.com/api/v5/users/octocat/repos",
        "events_url": "https://gitee.com/api/v5/users/octocat/events{/privacy}",
        "received_events_url": "https://gitee.com/api/v5/users/octocat/received_events",
        "type": "User",
        "site_admin": false
    }
}