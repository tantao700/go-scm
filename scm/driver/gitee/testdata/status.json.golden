{
    "State": 1,
    "Label": "default",
    "Desc": "the dude abides",
    "Target": "https://gitee.example.com/thedude/gitee-ce/builds/91"
}